KappaAnalysis/interface/Producers/ZProducer.h

```diff
@@ -95,6 +95,7 @@ class ZProducerBase : public KappaProducerBase
 	             KappaProduct& product,
 	             KappaSettings const& settings) const override
 {
+	LOG(WARNING) << "Correction of the jet collections will be applied!";
```

Comment:
What is this warning for?

Reply:
This indicates that the correction is applied on the collections.
It is somewhat deprecated and we don't use this anymore.

Reply:
removed